### ISSUE TITLE
HtmlUnitWebElement.getAttribute("outerHTML") returns null and "innerHTML" returns "outerHTML" value

### ISSUE THREAD

**Issue by stiemannkj1** [SeleniumHQ/htmlunit-driver#45]
`HtmlUnitWebElement.getAttribute("outerHTML")` returns `null`. Instead it should return the markup including the element's tag markup ([other browsers have `WebElement.getAttribute("outerHTML")`](http://stackoverflow.com/questions/32234205/how-to-get-html-code-of-a-webelement-in-selenium#32238342)).

## Steps to reproduce:

1. Add the following code to a Java file:

	```
	@Test
	public void testOuterHtml() {

		WebDriver webDriver = new HtmlUnitDriver(BrowserVersion.FIREFOX_45);
		webDriver.get("http://google.com");
		WebElement head = webDriver.findElement(By.xpath("//head"));
		System.out.println("-------------------------------------");
		System.out.println("innerHTML = ");
		System.out.println(head.getAttribute("innerHTML"));
		System.out.println("-------------------------------------");
		System.out.println("outerHTML = ");
		System.out.println(head.getAttribute("outerHTML"));
	}
	```

2. Run the test.

If the bug still exists, the following output will appear:

```
-------------------------------------
outerHTML = 
null
```

If the bug is fixed, the following output will appear:

```
-------------------------------------
outerHTML = 
<head>
<!-- ...snipped for brevity... -->
</head>
```

Related to this, it seems like `"innerHTML"` is actually implemented as `"outerHTML"` since it outputs the markup with the outer tags as well.

See:
- https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML
- https://developer.mozilla.org/en-US/docs/Web/API/Element/outerHTML

**Comment by stiemannkj1** [comment]
@asashour, not sure if this is able to be changed at this point, but looking at [the fix](https://github.com/SeleniumHQ/htmlunit-driver/commit/e2a2a89506bbafa0fa0dc6d0e3e83b5c895bf10d), I noticed that `innerHTML` was not fixed. `getAttribute("innerHTML")` should return html with only the children/text content. For example:

```
<div id="div">
    <button>button</button>
</div>
```

For the above html, `divElement.getAttribute("innerHTML")` should return `"<button>button</button>"`, but for HtmlUnit (at least `2.23`), it returns `"<div id="div"><button>button</button></div>"`. 

I'm not sure how long this bug has been there or if people rely on it, so maybe it cannot be fixed. But I figured I'd let you know about it.
